### PR TITLE
[14.0] [FIX] rename service context provider to avoid name clashes

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

**Review by kevinkhao — APPROVED** [review]

**Review by Mat-moran — APPROVED** [review]
I did not have any problem merging this PR, looks nice.

File: shopinvader_auth_api_key/components/service_context_provider.py [hunk 1]
```diff
@@ -12,10 +12,10 @@
 _logger = logging.getLogger(__name__)
 
 
-class AuthApiKeyServiceContextProvider(Component):
+class ShopinvaderAuthApiKeyServiceContextProvider(Component):
     _name = "auth_api_key.shopinvader.service.context.provider"
     _inherit = "shopinvader.service.context.provider"
-    _usage = "auth_api_key_component_context_provider"
+    _usage = "shopinvader_auth_api_key_component_context_provider"
```
Comment (simahawk) [inline shopinvader_auth_api_key/components/service_context_provider.py]: "component" seems redundant here
```suggestion
    _usage = "shopinvader_auth_api_key_context_provider"
```

Comment (PierrickBrun) [reply shopinvader_auth_api_key/components/service_context_provider.py]: Good idea

**Review by sebastienbeau — APPROVED** [review]
LGTM code review

**Pull request merged**